ui/login.go

```diff
@@ -16,23 +16,27 @@ const splashText = `
 
 type Login struct {
 	*tview.Flex
-	app          *tview.Application
-	tokenInput   *tview.InputField
-	loginButton  *tview.Button
-	tokenChannel chan string
-	messageText  *tview.TextView
-	runNext      chan bool
+	app                *tview.Application
+	tokenInput         *tview.InputField
+	tokenInputMasked   bool
+	tokenInputMaskRune rune
+	loginButton        *tview.Button
+	tokenChannel       chan string
+	messageText        *tview.TextView
+	runNext            chan bool
 }
 
 // NewLogin creates a new login screen with the login components hidden by default.
 func NewLogin(app *tview.Application, configDir string) *Login {
 	login := &Login{
-		Flex:         tview.NewFlex().SetDirection(tview.FlexRow),
-		app:          app,
-		tokenChannel: make(chan string, 1),
-		tokenInput:   tview.NewInputField(),
-		loginButton:  tview.NewButton("Login"),
-		messageText:  tview.NewTextView(),
+		Flex:               tview.NewFlex().SetDirection(tview.FlexRow),
+		app:                app,
+		tokenChannel:       make(chan string, 1),
+		tokenInput:         tview.NewInputField(),
+		tokenInputMasked:   true,
+		tokenInputMaskRune: 42,
```

Comment:
While runes are basically just numbers, you can put the character directly as long as you quote it: `'*'`. This is better, since not everyone has the ascii table in their head.